README.md

```diff
@@ -1,19 +1,22 @@
-iron-iconset
-============
-
-The `iron-iconset` element allows users to define their own icon sets.
-The `src` property specifies the url of the icon image. Multiple icons may
-be included in this image and they may be organized into rows.
-The `icons` property is a space separated list of names corresponding to the
-icons. The names must be ordered as the icons are ordered in the icon image.
-Icons are expected to be square and are the size specified by the `size`
-property. The `width` property corresponds to the width of the icon image
-and must be specified if icons are arranged into multiple rows in the image.
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-iconset.svg)](https://www.npmjs.com/package/@polymer/iron-iconset)
+[![Build status](https://travis-ci.org/PolymerElements/iron-iconset.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-iconset)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-iconset)
+
+## iron-iconset
+
+The `iron-iconset` element allows users to define their own icon sets.  The
+`src` property specifies the url of the icon image. Multiple icons may be
+included in this image and they may be organized into rows.  The `icons`
+property is a space separated list of names corresponding to the icons. The
+names must be ordered as the icons are ordered in the icon image.  Icons are
+expected to be square and are the size specified by the `size` property. The
+`width` property corresponds to the width of the icon image and must be
+specified if icons are arranged into multiple rows in the image.
 
 All `iron-iconset` elements are available for use by other `iron-iconset`
-elements via a database keyed by id. Typically, an element author that wants
-to support a set of custom icons uses a `iron-iconset` to retrieve
-and use another, user-defined iconset.
+elements via a database keyed by id. Typically, an element author that wants to
```

Comment:
This still has all of the old readme comments (that are automatically taken from the source). We are now just linking to the docs and using a simple example, as per https://raw.githubusercontent.com/PolymerElements/paper-input/85c6cfd45ba0f0c6c76bceee93b5f18890e2c655/README.md

Reply:
Fixed.